bot/cogs/doc.py

```diff
@@ -243,25 +305,39 @@ def __init__(self, bot: commands.Bot):
         if len(description) > 1000:
             shortened = description[:1000]
```

Comment:
It might be worth increasing this to 1500. Do we have cases where sigs can cover >500 characters? This would also ensure more complete information and doesn't seem too terribly lengthy to read. This is just a general suggestion to look into rather than a suggestion to change though.